tests/unit/test__pandas_helpers.py

```diff
@@ -57,13 +61,12 @@
 
 bigquery_storage = _versions_helpers.BQ_STORAGE_VERSIONS.try_import()
 
-PANDAS_MINIUM_VERSION = pkg_resources.parse_version("1.0.0")
+PANDAS_MINIUM_VERSION = "1.0.0"
 
-if pandas is not None:
-    PANDAS_INSTALLED_VERSION = pkg_resources.get_distribution("pandas").parsed_version
-else:
-    # Set to less than MIN version.
-    PANDAS_INSTALLED_VERSION = pkg_resources.parse_version("0.0.0")
+if pandas is not None:  # pragma: NO COVER
+    PANDAS_INSTALLED_VERSION = metadata.version("pandas")
+else:  # pragma: NO COVER
```

Comment:
Previously we didn't have `# pragma: NO COVER`. Is it needed?

Reply:
When I ran the kokoro build test it was giving me a failure on coverage in a test file (not this one), so I thought to add the tag everywhere just in case. It may have just been an issue with that test file, so I can remove it here and other places if needed!